Makefile

```diff
@@ -100,15 +100,30 @@ push-latest:
 	docker push $(IMAGE):latest-debug
 	docker push $(IMAGE):latest-rootless
 
+.PHONY: tag-edge
+tag-edge:
+	docker tag $(IMAGE):$(VERSION) $(IMAGE):edge
+	docker tag $(IMAGE):$(VERSION)-debug $(IMAGE):edge-debug
+	docker tag $(IMAGE):$(VERSION)-rootless $(IMAGE):edge-rootless
+
+.PHONY: push-edge
+push-edge:
+	docker push $(IMAGE):edge
+	docker push $(IMAGE):edge-debug
+	docker push $(IMAGE):edge-rootless
+
 .PHONY: docker-login
 docker-login:
 	@docker login -u ${DOCKER_USER} -p ${DOCKER_PASSWORD}
 
+.PHONY: image-full
+image-full: docker-login image-quick push
```

Comment:
Nit: this is super minor but I'd rather call the `image-full` target something like `push-image` to indicate that it's going to try to push. The other `image` and `image-quick` targets don't do any pushing (i.e., they just build locally).

Reply:
👍 updated in latest changes